style.css

```diff
@@ -0,0 +1,86 @@
+html,
+body {
+  margin: 0;
+  padding: 0;
+  width: 100%;
+  height: 100%;
+}
+
+body {
+  background-color: white;
+  color: #b13f6a;
+  text-align: center;
+  font-family: "Jost", sans-serif;
+  font-size: 20px;
+}
+
+.filter-bar {
+  text-align: left;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  padding: 20px;
+}
+
+.gallery-wrapper {
+  background-color: #b13f6a;
+  color: #ffb000;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: space-around;
+  padding: 20px;
+  box-sizing: border-box;
+}
+
+.card-wrapper {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  background-color: #ffb000;
+  color: #b13f6a;
+  width: calc(33.33% - 40px); /* 33.33% for 3 columns, subtracting margins */
```

Comment:
`calc() `enable more flexibility and precision in layout 👍 clear explanation by leaving comment